class-tgm-plugin-activation.php

```diff
@@ -18,8 +18,8 @@
  * Plugin Name: TGM Plugin Activation
  * Plugin URI:
  * Description: Plugin installation and activation for WordPress themes.
- * Version:     2.5.0
- * Author:      griffinjt, garyj, jrf
+ * Version:     2.5.0-RC
+ * Author:      Thomas Griffin, Gary Jones, Juliette Reinders Folmer
```

Comment:
Incorrect - the author names in the plugin headers are used on the Plugin list page - human readable.

WP.org usernames would only be needed for a readme.txt so that the repo links them correctly.